### ISSUE TITLE
Add a way to contact the author of the plugin

### ISSUE THREAD

**Issue by tomi2108** [trevorhauter/gitportal.nvim#80]
I would like to contact you @trevorhauter and I have no other means... may be this is a good opportunity to add a contact email to the readme or smth ?

**Comment by trevorhauter** [comment]
Hey @tomi2108 ! I think that's a great suggestion. I'll add my contact info to the readme. In the meantime, you can email me at trevorhauter99@gmail.com

**Comment by trevorhauter** [comment]
I was going to add my email to the readme but now I'm wondering if that will fill my inbox with a bunch of spam from scraping. How do plugins usually provide means of contact?

**Comment by tomi2108** [comment]
Yeah, I've looked around a bit and it does not look like it is a common thing. Maybe it is not as sensible as I initially tought.

**Comment by trevorhauter** [comment]
@tomi2108 I understand the need. I think there are good ways with google forms or a discord server, but not sure that this plugin warrants such at the moment. I'm going to close for now! Feel free to open another if it seems useful again or if we can think of a helpful way. I think also we may be able to use a chat room like neovim does? Although haven't looked into how that works..
